alacritty/src/display/mod.rs

```diff
@@ -351,8 +351,14 @@ impl Display {
             info!("Initializing glyph cache...");
             let init_start = Instant::now();
 
-            let cache =
-                renderer.with_loader(|mut api| GlyphCache::new(rasterizer, &font, &mut api))?;
+            let cache = renderer.with_loader(|mut api| {
+                GlyphCache::new(
+                    rasterizer,
+                    &font,
+                    &mut api,
+                    config.font.use_built_in_font_for_box_drawing,
```

Comment:
You don't need to pass it here, it's already stored on a `font` argument you're passing.